Library/OcBootManagementLib/ImageLoader.c

```diff
@@ -980,10 +980,9 @@ InternalEfiLoadImage (
                  );
     } else {
       //
-      // We verified the image, but contained garbage, or we are trying to secure boot a Fat slice.
-      // This should not happen.
+      // We verified the image, but contained garbage.
+      // This should not happen, just abort.
       //
-      ASSERT (FALSE);
```

Comment:
Why did you remove the `ASSERT`? Is this code reachable from now on?

Reply:
I am removing the assert because I added it - I don't think would do any harm to keep it, however. I can say with fair certainty that I added it so that if my added check ([see above](https://github.com/acidanthera/OpenCorePkg/pull/548#discussion_r1668144594)) was wrong, it would get flagged up sooner rather than later and be easy to identify...

Reply:
Just perhaps if it is an invariant, we should replace `Status` assignment with a `CpuDeadLoop()` call and keep the assert. If it is not, then your change makes sense.

Reply:
I don't think it is an invariant - we could get get to here without really verifying the file (when it's a dmg, as we are seeing in this issue), but then the file could generate an error when extracting a fat slice.

Reply:
Ok